app/src/main/java/org/groundplatform/android/model/submission/InstructionTaskData.kt

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright 2025 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.groundplatform.android.model.submission
+
+import kotlinx.serialization.Serializable
+
+@Serializable
+data class InstructionTaskData(val text: String) : TaskData {
```

Comment:
Are we planning to upload this task's value to remote storage? IMO, this is a read-only task and it doesn't make much sense to include this task in the submission mutation. If yes, we need to ensure that the value is skipped when uploading the data to remote storage.

@gino-m Wdyt?

Reply:
Can we add a unit test for this?

Reply:
removed this file

Reply:
I meant test to confirm that we don't upload the task's data to remote storage. See `SubmissionMutationConverterTest` for more reference.

Reply:
added test